### PR TITLE
Bump .NET SDK to 1.2.0.4 and fix tests

### PR DESCRIPTION
**Pull request by viovanov**

### CODE REVIEW

File: src/CloudFoundry.Build.Tasks.IntegrationTests/LoadYamlTest.cs [hunk 1]
```diff
@@ -3,18 +3,20 @@
...
 using System.IO;
 using System.Linq;
+using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace CloudFoundry.Build.Tasks.IntegrationTests
 {
     [TestClass]
+    [Ignore]
```
Comment (jenspinney) [inline src/CloudFoundry.Build.Tasks.IntegrationTests/LoadYamlTest.cs]: Why is this ignored?

Comment (jenspinney) [inline src/CloudFoundry.Build.Tasks.IntegrationTests/LoadYamlTest.cs]: (Talked offline -- yaml loading feature isn't working at the moment. This is being tracked separately).

**Comment by jenspinney** [comment]
LGTM

File: src/CloudFoundry.Build.Tasks.IntegrationTests/LoadYamlTest.cs (hunk 1 above)
Comment (viovanov) [inline src/CloudFoundry.Build.Tasks.IntegrationTests/LoadYamlTest.cs]: We don't have a proper manifest asset and the import from yml feature is WIP at the moment

**Pull request merged**